1_OCaml/TP/TP4.ipynb

```diff
@@ -9,7 +9,10 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        let f n= if n&gt;=0 then true
</pre><p>Au lieu de</p><p><code>&nbsp;&nbsp;let f n= if n&gt;=0 then true</code></p><p><code>&nbsp;&nbsp;&nbsp;&nbsp;else false in</code></p><p>Écrire :</p><p><code>let f n= n&gt;=0</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mp2i-fsm/mp2i-2021/pull/101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>